yarn.lock

```diff
@@ -267,10 +267,6 @@ ansi-html@0.0.7:
   version "0.0.7"
   resolved "https://registry.yarnpkg.com/ansi-html/-/ansi-html-0.0.7.tgz#813584021962a9e9e6fd039f940d12f56ca7859e"
 
-ansi-regex@*, ansi-regex@^3.0.0, ansi-regex@~3.0.0:
```

Comment:
Did you mean to include these changes to this PR?

Reply:
Yeah, that's fine. I think it's just some sub-dependencies that got updated.